message PBMsgReqUnassignHero
{
  required int64 cid = 1;
  required int64 hid = 2;
}
